### ISSUE TITLE
Can't delete message

### ISSUE THREAD

**Issue by mvb9** [modoboa/modoboa-webmail#27]
Hi, I have a problem with webmail. When user is full quota, he can't delete message, error display "Object object".  How to resolve problem? thank you very much.

**Comment by tonioo** [comment]
Hi, which version are you using ?

**Comment by mvb9** [comment]
My server using version 1.3.2 , in imaputils.py i was inserted "MAXLINE = 1000000
if hasattr(imaplib, "_MAXLINE") and getattr(imaplib, "_MAXLINE") < MAXLINE:
    setattr(imaplib, "_MAXLINE", MAXLINE)", quota user is 100 MB with less than 150 messages.

**Comment by tonioo** [comment]
And does it change something ?

**Comment by mvb9** [comment]
No any thing change.

**Comment by mvb9** [comment]
I find reason for this problem. When quota full function _cmd(self, name, *args, **kwargs) in imaputils.py return error. I was edit query user_query in dovecot-sql.conf.ext : user_query = SELECT '<mailboxes storage directory>/%d/%n' AS home, <uid> as uid, <gid> as gid, concat('*:bytes=', mb.quota, 'M') AS quota_rule, 'Trash:ignore' AS quota_rule2  FROM admin_mailbox mb INNER JOIN admin_domain dom ON mb.domain_id=dom.id WHERE mb.address='%n' AND dom.name='%d' . And I can move message to trash when quota full.

**Comment by yiminfantw** [comment]
I think this could still happen in 1.9.0 or later. My config is Ubuntu 16.04, Modoboa 1.9.1 and external LDAP. I had a test of this by reaching an user's quota and I can reproduce the "Object object" issue when deleting user email. Any idea? Thx

**Comment by tonioo** [comment]
@yiminfantw yes, I haven't officialized the proposed SQL query. You can try to apply it and see if it fixes the problem.
